enterprise_access/apps/bffs/handlers.py

```diff
@@ -69,29 +73,94 @@ def load_and_process(self):
         The method in this class simply calls common learner logic to ensure the context is set up.
         """
         try:
+            # Transform enterprise customer data
+            self.transform_enterprise_customers()
+
             # Retrieve and process subscription licenses. Handles activation and auto-apply logic.
-            # TODO: retrieve enterprise customer metadata,ENT-9629
-            # self.load_enterprise_customer()
             self.load_and_process_subscription_licenses()
 
-            # Retrieve default enterprise courses and enroll in the redeemable ones
-            self.load_default_enterprise_courses()
-            self.enroll_in_redeemable_default_courses()
+            # # Retrieve default enterprise courses and enroll in the redeemable ones
+            self.load_default_enterprise_enrollment_intentions()
+            self.enroll_in_redeemable_default_enterprise_enrollment_intentions()
         except Exception as e:  # pylint: disable=broad-exception-caught
             logger.exception("Error loading learner portal handler")
             self.add_error(
                 user_message="An error occurred while loading and processing common learner logic.",
                 developer_message=f"Error: {e}",
             )
 
+    def transform_enterprise_customers(self):
```

Comment:
TODO: i believe this is missing transforming the `staff_enterprise_customer` property